packages/camera-web/test/SimpleCameraHUD/SimpleCameraHUD.test.tsx

```diff
@@ -38,51 +40,33 @@ describe('SimpleCameraHUD component', () => {
     unmount();
   });
 
-  it('should display a take picture button that calls the handle.takePicture method', () => {
-    const handle = { takePicture: jest.fn() };
-    const { unmount } = render(<SimpleCameraHUD handle={handle} />);
-
-    fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-    expect(handle.takePicture).toHaveBeenCalled();
-    unmount();
-  });
-
-  it('should not throw any error when no handle is provided', () => {
-    expect(() => {
-      const { unmount, rerender } = render(<SimpleCameraHUD />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-
-      rerender(<SimpleCameraHUD handle={{}} />);
-      fireEvent.click(screen.getByTestId(TAKE_PICTURE_BTN_TEST_ID));
-      unmount();
-    }).not.toThrow();
-  });
-
```

Comment:
You are missing 3 tests here :

- One test that checks the onClick props of the TakePictureButton :
```typescript jsx
it('should pass the take picture callback from the handle to the TakePictureButton', () => {
  const handle = { takePicture: jest.fn() };
  const { unmount } = render(<SimpleCameraHUD handle={handle} />);

  expectPropsOnChildMock(TakePictureButtonMock,  { onClick: handle.takePicture });
  unmount();
});
```

- One test that checks the default onClick props of the TakePictureButton :
```typescript jsx
it('should not pass any onClick event to the TakePictureButton by default', () => {
  const { unmount } = render(<SimpleCameraHUD />);

  expectPropsOnChildMock(TakePictureButtonMock,  { onClick: undefined });
  unmount();
});
```

- One test that checks the size props of the TakePictureButton :
```typescript jsx
it('should set the size of the TakePictureButton to 60', () => {
  const handle = { takePicture: jest.fn() };
  const { unmount } = render(<SimpleCameraHUD />);

  expectPropsOnChildMock(TakePictureButtonMock,  { size: 60 });
  unmount();
});
```